src/components/SignUp.js

```diff
@@ -99,17 +100,19 @@ const initialValues = {
 };
 
 export default function SignUp() {
-  const handleSubmit = (event) => {
+  const handleSubmit = async (event) => {
     event.preventDefault();
     values.dob = dateOfBirth
       .toISOString()
-      .replace("-", "/")
       .split("T")[0]
-      .replace("-", "/");
-    DataService.creates(values);
-    console.log(values.dob);
-
+    values.password = await hash(values.password).toString()
+    //DataService.creates(values);
     console.log(values);
+    fet("/register", "POST", values).then(res => {
```

Comment:
can use both `http://localhost:6969/register` or just `/register`  works the same